pkg/config/config_template.yaml

```diff
@@ -4109,6 +4109,15 @@ api_key:
     #
     # enabled: true
 
+    ## @param compute_top_level_by_span_kind - boolean - optional - default: true
+    ## @env DD_OTLP_CONFIG_TRACES_COMPUTE_TOP_LEVEL_BY_SPAN_KIND - boolean - optional - default: true
+    ## [BETA] Enables new logic to improve top-level spans identification in OTLP ingest.
```

Comment:
Given this is a BETA feature, is there ANY chance this name might change or that this config might be removed? If so we should consider using the APM Features `DD_APM_FEATURES` which makes removing, or renaming features easier (and to me makes it a bit clearer that this feature is NOT stable and is allowed to change)

Reply:
Yes there's a chance that this config might be removed once the feature is considered stable. We want this config option to be enabled by default though, is this possible with APM Features?

Reply:
🤔 this is a bit odd to me and I must have missed this in my first read-through. The feature is considered "beta" but it will be enabled by default? (Technically this is possible, it would just mean adding an APM Feature that "disables" this new default behavior)

But I have to say enabling a "beta feature" by default sounds like a recipe for disaster to me. I'm not sure why this change is considered beta (are there known situations where enabling this will break setups with no workaround yet? Are there still changes to make somewhere in the backend?), but I would say that we should either resolve those (so this isn't called "beta") OR we shouldn't enable the change by default

Reply:
The main reason why we want it to be enabled by default is that the existing logic for detecting top-level spans for OTel is broken, and this update should fix that behavior and be more consistent. I marked it as "beta" because it's going to be a breaking change and there may be future changes to use this top-level logic beyond OTel spans as well ([see roadmap](https://docs.google.com/document/d/1ztc9lMB6mNsUCPT5gSU56MuQ0p6x3hGhu8ptP--Kxfk/edit#heading=h.d06p0pozwn2a)).

There aren't any more changes to be made in the backend and there are no known situations where enabling this will break setups with no workaround, so I could just be misunderstanding the classification of "beta". I've also tested this feature to verify that it works correctly so we don't have to mark it as "beta" if that makes more sense

Reply:
Just updated my PR to use APM Features instead of a new config option. @ajgajg1134 and I discussed offline and decided that this feature doesn't need to be marked as "beta" because it's fixing a known issue with top-level identification. We are still adding this under APM Features to make it easier for future changes based on the proposed roadmap.